### DISCUSSION TITLE
Strange error `ESLint couldn't determine the plugin "@typescript-eslint" uniquely`

### DISCUSSION THREAD

**Discussion by helltraitor** [typescript-eslint/typescript-eslint#7303] [Community Feedback]
I've change nothing (maybe dependencies are update occured when `yarn.lock` was removed), and when I tried to execute the release script (includes `eslint .` command). I've received the following error:

```
Oops! Something went wrong! :(

ESLint: 8.42.0

ESLint couldn't determine the plugin "@typescript-eslint" uniquely.

- S:\Programming\GitHub\helltraitor\nuxt-feedme\node_modules\@typescript-eslint\eslint-plugin\dist\index.js (loaded in ".eslintrc » @antfu/eslint-config » @antfu/eslint-config-vue » @antfu/eslint-config-ts » plugin:@typescript-eslint/recommended » ./configs/base")
- S:\Programming\GitHub\helltraitor\nuxt-feedme\node_modules\@nuxt\eslint-config\node_modules\@typescript-eslint\eslint-plugin\dist\index.js (loaded in ".eslintrc » @nuxt/eslint-config")

Please remove the "plugins" setting from either config or remove either plugin installation.

If you still can't figure out the problem, please stop by https://eslint.org/chat/help to chat with the team.
```

Error can be fixed if comment the next line...
https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/src/configs/base.ts#L11

I don't understand what I'm doing wrong (or maybe this is rare issue, idk, really)

My config is
```
// .eslintrc
{
  "root": true,
  "extends": [
    "@nuxt/eslint-config",
    "@antfu"
  ],
  "rules": {
    "antfu/top-level-function": 0
  }
}
```

Any help appreciated. Since this is not critical for me, I just comment this line (in dist folder of node_modules), but I would like to resolve this issue

**Comment by bradzacher** [comment]
As per our contributing guides - we do not use discussions for support.

Please see our contributing guide for alternatives.

https://typescript-eslint.io/contributing/discussions
